vim.html.markdown

```diff
@@ -0,0 +1,231 @@
+---
+category: tool
+tool: vim
+contributors:
+    - ["RadhikaG", "https://github.com/RadhikaG"]
+filename: LearnVim.txt
+---
+
+
+[Vim](www.vim.org)
+(Vi IMproved) is a clone of the popular vi editor for Unix. It is a a text 
+editor designed for speed and increased productivity, and is ubiquitous in most 
+unix-based systems. It has numerous keybindings for speedy navigation to 
+specific points in the file, and for fast editing.
+
+## Basics of navigating Vim
+
+```
+    vim <filename>   # Open <filename> in vim
+    :w               # Save current file
+    :q!              # Quit vim without saving file
```

Comment:
Explain it a little more I mean. Instead of saying "quit vim without saving file" could explain that the `!` character forces the `:q` command to run which is quit leading to vim exiting without saving.